hud.tscn

```diff
@@ -14,15 +14,19 @@ anchor_left = 0.5
 anchor_top = 0.5
 anchor_right = 0.5
 anchor_bottom = 0.5
-offset_left = -194.5
-offset_top = -396.0
-offset_right = 194.5
-offset_bottom = -217.0
+offset_left = -367.5
+offset_top = -487.0
+offset_right = 367.5
+offset_bottom = 56.0
 grow_horizontal = 2
 grow_vertical = 2
 size_flags_horizontal = 4
 theme_override_font_sizes/font_size = 64
-text = "Press any key
+text = "Controls
+Player left: WASD
+Player right: Arrow keys
```

Comment:
I like the "player left" / "player right" scheme here, but it's awkward in the default case where we have a single player node. I'd be interested if anyone has an alternative scheme that works better, but with a similar bit of clarity over "player 1" / "player 2"?

Reply:
Good point, Player 1 / Player 2 is better.